src/hypothesis/searchstrategy/regex.py

```diff
@@ -27,23 +27,19 @@
 from hypothesis import reject
 from hypothesis.internal.compat import PY3, hrange, hunichr, text_type, \
     int_to_byte
+from hypothesis.internal.charmap import categories
 
 HAS_SUBPATTERN_FLAGS = sys.version_info[:2] >= (3, 6)
 
 
-UNICODE_CATEGORIES = set([
-    'Cf', 'Cn', 'Co', 'LC', 'Ll', 'Lm', 'Lo', 'Lt', 'Lu',
-    'Mc', 'Me', 'Mn', 'Nd', 'Nl', 'No', 'Pc', 'Pd', 'Pe',
-    'Pf', 'Pi', 'Po', 'Ps', 'Sc', 'Sk', 'Sm', 'So', 'Zl',
-    'Zp', 'Zs',
-])
+UNICODE_CATEGORIES = set(categories())
 
 
 SPACE_CHARS = set(u' \t\n\r\f\v')
 UNICODE_SPACE_CHARS = SPACE_CHARS | set(u'\x1c\x1d\x1e\x1f\x85')
 UNICODE_DIGIT_CATEGORIES = set(['Nd'])
 UNICODE_SPACE_CATEGORIES = set(['Zs', 'Zl', 'Zp'])
-UNICODE_LETTER_CATEGORIES = set(['LC', 'Ll', 'Lm', 'Lo', 'Lt', 'Lu'])
```

Comment:
What's going on here exactly? Were some of these not correct?

Reply:
`'LC'` isn't a category, and I put them in normative order while I was there.